lib/components/gene/header.js

```diff
@@ -46,7 +48,11 @@ class Header extends React.Component {
         </View>
       )
     } else {
-      return (<View style={styles.followButton}/>)
+      return (
+        <View style={styles.followButton}>
+          <InvertedButton text="Follow" textOpacity={0.5} />
+        </View>
```

Comment:
adds the button, even if it's not got follow data yet, was hard to work with the gap ATM